readthedocs/vcs_support/backends/svn.py

```diff
@@ -58,7 +58,11 @@ def up(self):
     def co(self, identifier=None):
         self.make_clean_working_dir()
         if identifier:
-            url = self.base_url + identifier
+            base = self.base_url.rstrip('/')
+            tag = ''
+            if identifier:
```

Comment:
Added this if-condition because if `identifier` is `None`, then the `lstrip()` would throw an error.